news/2017-10-20-scorewatch-october-week-1.md

```diff
@@ -0,0 +1,53 @@
+
+![](/wiki/shared/news/2017-10-20-scorewatch-october-week-1/banner.jpg)
+
+[Bubbleman](https://osu.ppy.sh/u/5182050) is taking his limits to the next level as he snipes a year old score on [MY FIRST STORY - ALONE](https://osu.ppy.sh/b/861381), taking an amazing 99.91% HDHR FC to grab 527pp. To think that he nearly SSed it (got the 1x 100 on the 2nd last note) is amazing enough!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/yWXkzn1roPE" frameborder="0" allowfullscreen></iframe></br>
+
+[Rafis](https://osu.ppy.sh/u/2558286) has sniped hvick225’s year old score on [Hana - Sakura no Uta](https://osu.ppy.sh/b/827488)! This is Rafis’ 11th 700pp score as he sets an impressive 99.84% HDDT FC on the map, earning 704pp. Get ready for some intense jumping and streaming!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/wdTxovl8rMU" frameborder="0" allowfullscreen></iframe></br>
+
+[Cookiezi](https://osu.ppy.sh/u/124493) has reached the 14,000pp mark with an amazing 99.25% HDHR 1 miss score on [Linkin Park - Guilty All The Same (feat. Rakim)](https://osu.ppy.sh/b/1187302?m=0) to earn 726pp. Enjoy 6 minutes of Linkin Park!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/HDfWRLlqhY8" frameborder="0" allowfullscreen></iframe></br>
+
+[cheewee10](https://osu.ppy.sh/u/4477497) sniped WindyS to the number one spot on [gmtn. (witch's slave) - furioso melodia](https://osu.ppy.sh/b/687730) on the 7K difficulty with a painful 99.99% (1 200!) SD FC, earning himself 390pp. Imagine if he put the PF mod instead.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/1DMQ2gcQD7c" frameborder="0" allowfullscreen></iframe></br>
+
+
+BLAST FROM THE PAST
+-----
+
+[rustbell](https://osu.ppy.sh/u/227717) doing what he does best. His 94.78% HDDT FC on DJPop’s [Yuyoyuppe feat. Hanatan - Palette](https://osu.ppy.sh/s/15951) is pretty unbelievable. 270BPM bursts, high note density, and no one has even come close to his score set roughly a year ago. 
+
+Once again, do support [osu!archive](https://www.reddit.com/r/osugame/comments/75ev3d/osuarchive_44_rustbell_yuyoyuppe_feat_hanatan/) (it is still ongoing on the osugame reddit), more scores are being churned out as this gets published! 
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/cRHOi5cqzsQ" frameborder="0" allowfullscreen></iframe></br>
+
+NOTABLE MENTIONS
+-----
+
+[Adamqs](https://osu.ppy.sh/u/613592) has taken an impressive 99.44% 2x miss FL score on [Hana - Sakura no Uta](https://osu.ppy.sh/b/827488)! Currently, he is the only player to ever clear the map with FL. There have been many FL plays in osu! history, but this is definitely one of the craziest we will see.
+
+SCORESHOW WITH ZSEIKIMATSU
+-----
+
+[\_yu68](https://osu.ppy.sh/u/6170507) and [asuasu_yura](https://osu.ppy.sh/u/2875968) take on each other on the recently ranked osu!taiko marathon, [kamome sano - </emotional>](https://osu.ppy.sh/b/1105943).
```

Comment:
use a different title or something because the coding makes the word invisible lol

Reply:
more like they need to unescape every char ``(\<\/emotional\>)``

good catch tho